### ISSUE TITLE
Jar Design A320 3.2r1 - xplane 11.10b7

### ISSUE THREAD

**Issue by zippy77777** [skiselkov/BetterPushbackC#171]
in 0.44 version on BP all works beautifully :-)

in 0.45 (latest) when BP connects to plane it shuts down packs and generators and engine power down  until the tug releases.

**Comment by dcrowe24** [comment]
I have exactly the same issue :( Only when tug disconnected and drove off did my engines suddenly kick in to life. The max N1 I would get is about 5.0 when connected tot the tug.

**Comment by zippy77777** [comment]
Yeah same with the 5%

Sent from my iPhone

On 14 Nov 2017, at 12:35, dcrowe24 <notifications@github.com<mailto:notifications@github.com>> wrote:


I have exactly the same issue :( Only when tug disconnected and drove off did my engines suddenly kick in to life. The max N1 I would get is about 5.0 when connected tot the tug.

**Comment by hansw1204** [comment]
Same here.

**Comment by skiselkov** [comment]
I believe the JAR may be using the throttle control in some weird fashion. If you happen to have DatarefEditor or DatarefTool, can you check what value `sim/flightmodel/engine/ENGN_thro_use` is set to when the aircraft is idling?

**Comment by zippy77777** [comment]
Just downloaded datarefeditor and going into the sim to check for you

**Comment by zippy77777** [comment]
use=(0/8) 0.16****   the first * is either a 0 or a 1 and the last 3 are constantly moving!

**Comment by zippy77777** [comment]
when I say first * i mean the one next to the 6

**Comment by skiselkov** [comment]
I see, so the values are 0.16, 0.16 when at idle (with no pushback attached). Seems the JAR might be using a non-zero throttle value at idle. Normally ENGN_thro_use should be 0 when at idle. I will incorporate a workaround for the JAR aircraft.

**Comment by zippy77777** [comment]
Your a very nice bloke ;-) thank you chap

**Comment by skiselkov** [comment]
Can you please test with BP 0.46? I've removed the throttle control entirely.

**Comment by zippy77777** [comment]
Yeah I have just tried the new one on the site and it seems to work ok. At one stage I did hot on the developer bit at the bottom which crashed the whole sim back to the desktop though, thought you might be interested.

Also, is there a way of getting the default chap to do the audio as I can’t seem to find him anymore?

And thank for all your hard work 😊

Darren...



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by Rodeo314** [comment]
Instead of overriding throttles and commanding a zero throttle value, you could probably set throttle ratio all (whatever that dataref is actually called) or something similar, which usually affects the levers themselves even if the aircraft has code to override throttles.

**Comment by Rodeo314** [comment]
But yeah, since you’re not overriding anything you’d need to either call it repeatedly or set it once and live with the fact that users may have noisy hardware…
